src/integration/README.md

```diff
@@ -14,38 +14,91 @@ This will build and run the application that the tests will run against.
 
 ## How to write tests
 
-An example of a test that would check a headline renders in the document on both AMP and canonical platforms would look like this:
+We need to write tests for both our AMP and canonical platforms. There are tests that can be written to test both platforms (cross platform tests), some tests for canonical only, and some tests for AMP only.
 
-```js
-/**
- * @pathname /mundo/articles/ce42wzqr2mko
- */
+#### Cross platform tests
+
+For tests that can be run in both platforms, we should add these tests to a file called `crossPlatformTests.js`. An example of a cross platform test would be like
 
-[amp, canonical].forEach((page) => {
+```js
+export default () => {
   it('I can see the headline', () => {
-    const headlineEl = page.getByText(headlineText);
+    const h1El = document.querySelector('h1');
 
-    expect(headlineEl).toBeInTheDocument();
+    expect(h1El).toBeInTheDocument();
+    expect(h1El).toBeTruthy();
```

Comment:
Should this be `expect(h1El.textContent).toBeTruthy();` (just looking at other patterns across the tests, and we seem to check that the text, rather than the component itself is truthy